### ISSUE TITLE
attempt to add with overflow

### ISSUE THREAD

**Issue by inaridiy** [InternetMaximalism/intmax-rollup-cli#27]
When I tried to check my balance after minting a large amount, I received an Overflow Error.

The original text of the error is as follows
```
$ intmax assets
thread 'main' panicked at 'attempt to add with overflow', src/utils/key_management/types.rs:77:17
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

## Generating Environment
Macbook M1Pro
DevContainer
```
Linux ccb1ef3cc310 5.10.104-linuxkit #1 SMP PREEMPT Thu Mar 17 17:05:54 UTC 2022 aarch64 GNU/Linux
```

**Comment by hrmk1o3** [comment]
Thank you for trying this CLI. This problem fixed the following commit: b52618451bdb439214d62d8a7fb6be84bc054d07 .
As the problem is simply a display fault, you can use this CLI after updating it to the latest state.

**Comment by inaridiy** [comment]
Thank you very much.
I was able to check my balance successfully.
Sorry for the trouble...
